src/setData/dlc1.js

```diff
@@ -748,11 +748,21 @@ module.exports = [
           {
             title: 'Shaders',
             items: [
-              167651268, // Crimson Passion
-              2112889975, // Crimson Valor
-              1052553862, // Crimson Valor
+              1052553863, // Crimson Passion - Bright Dust dismantle
+              1052553862, // Crimson Valor - Bright Dust dismantle
+              167651268, // Crimson Passion - Legendary Marks dismantle
```

Comment:
Ahhh good point. If the items don’t drop, there’s not really much use in having these show up then. 

If you get a moment would you mind submitting a new PR to remove the unobtainable items?

Reply:
Man, it’s moments like this where having stats on how many people have collected what would be really useful...

Reply:
The new PR is 59.

Yeah, those stats would be helpful. I was only able to figure it out, because I participated and had the two that dropped with the newer ID numbers.